lm/lm_image.py

```diff
@@ -0,0 +1,86 @@
+__all__ = ["LmImageJob"]
+
+import shutil
+
+from sisyphus import *
+from i6_core import rasr
+from i6_core import util
+
+Path = setup_path(__package__)
+
+
+class LmImageJob(rasr.RasrCommand, Job):
+    """
+    pre-compute LM image without generating global cache
+    """
+
+    def __init__(
+        self,
+        crp,
+        extra_config=None,
+        extra_post_config=None,
+        encoding="utf-8",
+        mem=2,
+    ):
+        kwargs = locals()
+        del kwargs["self"]
+
+        self.config, self.post_config = LmImageJob.create_config(**kwargs)
+        self.exe = self.select_exe(crp.lm_util_exe, "lm-util")
+
+        self.log_file = self.log_file_output_path("lm_image", crp, False)
+        self.out = self.output_path("lm.image", cached=True)
```

Comment:
In my personal opinion, the output member should have a more descriptive name than simply `out`; e.g. `out_image` or `out_lm_image`